region/maridia/outer/Mama Turtle Room.json

```diff
@@ -886,6 +903,46 @@
       ],
       "flashSuitChecked": true,
       "note": "To save energy, start from the upper right ledge; it may require repositioning Mama Turtle to the right to get to the ledge."
+    },
+    {
+      "link": [5, 6],
+      "name": "Base",
+      "requires": [
+        "canUseGrapple",
```

Comment:
^^ NB: is grappling back across okay "base" when Mama Turtle is awake?  The 5->2 strat is only for Mama asleep, but getting on top of the item is significantly trickier than simply crossing the room.

Reply:
Okay, so in testing this... this base grapple from ledge is slightly tricky as the block is barely offscreen at the moment you have to fire grapple.  It's a lot easier if you have HiJump or Walljump.